### PR TITLE
Adding note regarding logging support (BSC#1174631)

### PR DESCRIPTION
**Pull request by asettle**

### CODE REVIEW

File: xml/bp_troubleshooting_cephadm.xml [hunk 1]
```diff
@@ -60,6 +60,10 @@
  </sect1>
  <sect1 xml:id="bp-troubleshooting-cephadm-logs">
    <title>Accessing &ceph; Daemon Logs</title>
+   <para>
+     &productname; &productnumber; supports &ceph; logging via systemd-journald. To access
+     the logs of &ceph; daemons in  &productname; &productnumber;, follow the next instructions.
```
Comment (asettle) [inline xml/bp_troubleshooting_cephadm.xml:65]: Follow the proceeding instructions? Hm, @lproven what do you think?

Comment (smithfarm) [reply xml/bp_troubleshooting_cephadm.xml:65]: "To learn how to access the logs of Ceph daemons, read the following text."

"The procedure for accessing the logs of Ceph daemons is described below."

(could probably think of more?)

Comment (asettle) [reply xml/bp_troubleshooting_cephadm.xml:65]: Described below follows more along with what we'd normally say :thinking:  my brain keeps going "follow the following instructions" like it's on repeat

Comment (smithfarm) [reply xml/bp_troubleshooting_cephadm.xml:65]: "follow the instructions, below" is another option

Comment (smithfarm) [reply xml/bp_troubleshooting_cephadm.xml:65]: or "use the following procedure"

**Review by LenzGr — APPROVED** [review]
LGTM!

**Review by smithfarm — APPROVED** [review]

**Pull request merged**